용재/1874.py

```diff
@@ -0,0 +1,33 @@
+# https://www.acmicpc.net/problem/1874
+# 스택수열
+
+N = int(input())
+
+inputNumbers = [int(input()) for _ in range(N)]
+
+stack = []
+answer = []
+
+index = 0
+
+flag = False
+
+for each in inputNumbers:
+  while True:
+    if len(stack) == 0 or stack[-1] != each:
```

Comment:
stack의 길이가 0이라는 것은 아래와 같이 나타낼 수 있을 것 같아요!

```suggestion
    if not stack or stack[-1] != each:
```